app/connectors/SubmittedProcessConnector.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.AppConfig
+import connectors.httpParsers.SaveSubmittedProcessHttpParser.saveSubmittedProcessHttpReads
+import javax.inject.{Inject, Singleton}
+import models.{RequestOutcome, SaveSubmittedProcessResponse}
+import play.api.libs.json.JsValue
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SubmittedProcessConnector @Inject()(httpClient: HttpClient, appConfig: AppConfig) {
```

Comment:
If we separate out the approval into its own connector, why not call it ApprovalConnector with initial call submitForApproval(). Although separating out does leave the existing GuidanceConnector which submits scratch processes, misnamed

Reply:
Hi Chris, I decided the connector would be for the submittedProcesses and approval is just one thing that can be done to them.  I am assuming we will have submittedProcesses with various status changes that all go through this connector

Reply:
The existing GuidanceConnector also submits a process, the scratch process

Reply:
ScratchProcesses and SubmittedProcesses are two separate processes - the submit being done for Scratch is as an action, so I was just keeping separation like we do for controller/service/repo.  Happy to put them in to central connector, but separation means less likelihood of impacting other code if changes made to one or other.

Reply:
I'd tend to agree with Chris here. The term guidance is all encompassing; all processes can be considered to be guidance. In the view project there is only a guidance connector.

Reply:
ok, will move the submittedProcess connector into the central GuidanceConnector, thanks